pybamm/expression_tree/functions.py

```diff
@@ -446,6 +450,10 @@ def _function_diff(self, children, idx):
         """See :meth:`pybamm.Function._function_diff()`."""
         return Cosh(children[0])
 
+    def _sympy_operator(self, child):
+        """Override :meth:`pybamm.Function._sympy_operator`"""
+        return sympy.sinh(child)
```

Comment:
could have a default behavior (e.g. in SpecificFunction._sympy_operator) for functions that have the same name in pybamm and sympy:
```python3
class_name = self.__class__.__name__.lower()
sympy_function = getattr(sympy, class_name)
return sympy_function(child)
```